CHANGELOG-developer.next.asciidoc

```diff
@@ -138,6 +138,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Add gcp dataproc metricset. {pull}30008[30008]
 - Add Github action for linting
 - Add regex support for drop_fields processor.
+- Improve stability and flakyness of Python tests {pull}31588[31588]
```

Comment:
[Suggestion]
I don't think you want to improve _flakyness_ :)
```suggestion
- Improve stability reducing flakyness of Python tests {pull}31588[31588]
```

Reply:
I've improved the wording.